src/View/Helper/FlashMessenger.php

```diff
@@ -369,4 +288,32 @@ protected function getEscapeHtmlHelper()
 
         return $this->escapeHtmlHelper;
     }
+
+    /**
+     * @return array<string, FlashMessengerNamespace>
+     */
+    public function getNamespaces(): array
+    {
+        return $this->namespaces;
+    }
+
+    /**
+     * @return FlashMessenger
+     */
+    public function addNamespace(FlashMessengerNamespace $flashMessengerNamespace): self
+    {
+        $this->namespaces[$flashMessengerNamespace->getName()] = $flashMessengerNamespace;
+
+        return $this;
+    }
+
+    public function getDefaultNamespace(): FlashMessengerNamespace
+    {
+        return $this->defaultNamespace;
+    }
+
+    public function setDefaultNamespace(FlashMessengerNamespace $defaultNamespace): void
+    {
+        $this->defaultNamespace = $defaultNamespace;
+    }
```

Comment:
We should probably limit the state mutation to the constructor as much as possible, rather than allowing further runtime modification (and therefore complexity)